.gitignore

```diff
@@ -52,8 +52,17 @@ terraform/testnet/*-kubernetes.json
 terraform/testnet/*-vault.ca
 .terraform.lock.hcl
 
-# Move Build Output
-build/
+# Move build output
+aptos-move/framework/aptos-framework/build/*
+aptos-move/framework/aptos-stdlib/build/*
+aptos-move/move-examples/data_structures/build/*
+
+# Included Move build output
+!aptos-move/framework/aptos-framework/build/AptosFramework
+aptos-move/framework/aptos-framework/build/AptosFramework/*
+!aptos-move/framework/aptos-framework/build/AptosFramework/docs
+aptos-move/framework/aptos-framework/build/AptosFramework/docs/*
+!aptos-move/framework/aptos-framework/build/AptosFramework/docs/storage_gas.md
```

Comment:
Having specifics here is kinda weird.  Is this just because we want this doc in the AptosFramework/docs folder?

Reply:
> 

Correct: @davidiw mentioned previously that automatic docgen broke the CI system, and since the [framework-docs](https://github.com/aptos-labs/aptos-core/tree/framework-docs) branch is 4 months stale at this point, it looks like there is no system for making sure docgen files are up-to-date. Hence the specific ignore override patterns above, which only preserves the internally cross-linked `storage_gas.md` docgen file for this commit.

Keeping the file in the `AptosFramework/docs` folder is intended to simplify the docgen building process such that it is kept up-to-date in the future, hence the associated build scripting file below: allowing the docgen file to be tracked in its native build location with a simple script call reduces friction and makes it easier to keep the docgen file current against source.

Reply:
The framework should be very stable at this point in time. Let's move this content over there.